Modify the initialization code of waveshare st77916 1.85

ggc121238 opened the pull request:
屏幕厂家有改动，需要适配新的初始化代码

Lzw655 commented on src/drivers/lcd/port/esp_lcd_st77916.c:
Hi @ggc121238,

请直接修改配置文件中的 [ESP_PANEL_BOARD_LCD_VENDOR_INIT_CMD](https://github.com/esp-arduino-libs/ESP32_Display_Panel/blob/4c23160bddefe70be36d14fd3972834a45169c74/esp_panel_board_custom_conf.h#L293) 来配置 LCD 参数，无需修改 *esp_lcd_st77916.c* 中的参数。

ggc121238 committed:
Modify ESP32_S3_TOUCH_LCD_1_85.h
Initialize the screen code for the development board and correct the COLOR INEVRT
